BadHabbit/BadHabbit/UserStories/NewsScreen/NewsScreenViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  NewsScreenViewController.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 28/05/2023.
+//
+
+import UIKit
+
+final class NewsScreenViewController: UIViewController {
+    
+    private let viewModel: ExampleViewModelProtocol
+    
+    let textLabel = UILabel()
+    let errorLabel = UILabel()
+    
+    init(viewModel: ExampleViewModelProtocol) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupUI()
+        bindVM()
+    }
+    
+    deinit {
+        print("dinit)")
+    }
+}
+
+extension NewsScreenViewController {
+    private func loadDataButtonConfigure() -> UIButton {
+        let loadDataButton = UIButton()
+        loadDataButton.translatesAutoresizingMaskIntoConstraints = false
+        loadDataButton.setTitle("Load news", for: .normal)
+        loadDataButton.backgroundColor = .systemGreen
+        
+        loadDataButton.addTarget(self, action: #selector(loadButtonTapped), for: .touchUpInside)
+        loadDataButton.tag = 1
```

Comment:
what is tag for ?

Reply:
If you need to access this view out of this function - just make property not local but class property